data/mods/Aftershock/mobs/aliens.json

```
@@ -0,0 +1,155 @@
+[
+  {
+    "id": "afs_mon_rabbit",
+    "type": "MONSTER",
+    "name": { "str": "labbit" },
+    "description": "A rabbit from earth, genetically modified to eat the lichen of Salus IV.  It's fur is rust colored and its ears are shortened to preserve warmth.",
+    "default_faction": "small_animal",
+    "bodytype": "pig",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "1600 ml",
+    "weight": "1600 g",
+    "hp": 8,
+    "speed": 160,
+    "material": [ "flesh" ],
+    "symbol": "r",
+    "color": "brown",
+    "aggression": -99,
+    "morale": -7,
+    "melee_cut": 0,
+    "dodge": 6,
+    "reproduction": { "baby_monster": "afs_mon_rabbit", "baby_count": 3, "baby_timer": 55 },
+    "harvest": "alien_small_fur",
+    "fear_triggers": [ "SOUND", "PLAYER_CLOSE" ],
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "CATTLEFODDER", "PET_WONT_FOLLOW", "WARM" ]
+  },
+  {
+    "id": "afs_runner",
+    "type": "MONSTER",
+    "name": { "str_sp": "runner" },
+    "description": "A large fluffy three-legged herbivore, extremely fast and cowardly.  It usually grazes alone, and relies on its keen senses to stay safe.",
+    "default_faction": "herbivore",
+    "bodytype": "horse",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "386000 ml",
+    "weight": "386 kg",
+    "hp": 120,
+    "speed": 200,
+    "material": [ "flesh" ],
+    "symbol": "H",
+    "color": "brown",
+    "aggression": -99,
+    "morale": -7,
+    "melee_cut": 0,
+    "dodge": 1,
+    "armor_bash": 4,
+    "armor_cut": 1,
+    "armor_bullet": 1,
+    "vision_night": 25,
+    "path_settings": { "max_dist": 10 },
+    "fear_triggers": [ "SOUND", "PLAYER_CLOSE" ],
+    "death_function": [ "NORMAL" ],
+    "harvest": "alien_large_leather",
+    "biosignature": { "biosig_item": "feces_manure", "biosig_timer": 8 },
+    "flags": [ "SEES", "HEARS", "SMELLS", "PET_MOUNTABLE", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM" ]
+  },
+  {
+    "id": "afs_grazer",
+    "type": "MONSTER",
+    "name": { "str_sp": "grazer" },
+    "description": "A large furred herbivore, with its heavy head constantly against the ground.  It uses it clawed hooves to shovel up the snow to reach any dead plant life to eat.",
+    "default_faction": "herbivore",
+    "bodytype": "horse",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "386000 ml",
```

```suggestion
    "volume": "386 L",
```
I could be wrong on the suggested format but I was confused by the massive amount of ml